src/db/ContentGenerator.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace src\db;
+
+use PDO;
+
+class ContentGenerator
+{
+    const LETTER_PATTERN = "/[a-zA-Z]/";
+
+    private array $sortNames = ['name', 'code', 'state', 'city'];
+
+    public function __construct(private Paginator $paginator)
+    {
+    }
+
+    public function request(array $data): array
+    {
+        $filters = $this->validateRequest($data);
```

Comment:
Please move call of `validateRequest` method inside of `getAirports` and remove `request` method